### PR TITLE
Login branch

### PR DESCRIPTION
**Pull request by mlynch5187**
This branch deals with logging in through Google.

- Omniauth allows logging in through Google authorization
- Sessions controller and required routes created
- Test needs to be written for Omniauth
- Tests 100% passing at 14.29% covered
- Rubocop at 100%

### CODE REVIEW

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class SessionsController < ApplicationController
+  def create
```
Comment (IamNorma) [inline app/controllers/sessions_controller.rb:2]: Nice work with this method

File: config/initializers/omniauth.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+Rails.application.config.middleware.use OmniAuth::Builder do
+  provider :google_oauth2, ENV['GOOGLE_CLIENT_ID'], ENV['GOOGLE_CLIENT_SECRET'],
+    {
+      scope: 'userinfo.email, calendar',
```
Comment (IamNorma) [inline config/initializers/omniauth.rb:4]: Might have to specify scope for calendar.events

File: config/routes.rb [hunk 3]
```diff
@@ -2,4 +2,8 @@
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
 
   root 'welcome#index'
+
+  # get 'auth/:provider/callback', to: 'sessions#googleAuth'
+  get '/auth/:provider/callback', to: 'sessions#create'
```
Comment (IamNorma) [inline config/routes.rb:7]: Nice work with changing this to a RESTful route

File: app/controllers/sessions_controller.rb (hunk 1 above)
Comment (mlynch5187) [reply app/controllers/sessions_controller.rb:2]: Thanks! I figured it out entirely on my own.

**Pull request merged**